public/fav/manifest.json

```diff
@@ -1,41 +1,41 @@
 {
-    "name": "Blue Lupi Homepage",
```

Comment:
Why has it been changed to "name": "App"?

Reply:
Ah yes. This file has been auto generated.